pytest-selenium/utils/utility.py

```diff
@@ -11,3 +14,16 @@ def books(self) -> [str]:
 
     def random_book_slug(self):
         return random.choice(self.books)
+
+
+class WaitForTitleChange(object):
+    def click_and_wait_for_load(self, element: WebElement = None) -> None:
```

Comment:
starting to get why you did this, because you're calling `click_and_wait_for_load` from both regions and pages, so you define `page_title` in page, and then you have a helper in the base region to also do `page_title`.

seems like it'd be overall simpler to have a helper in the base region to find the root page (called like, `page` or whatever), and then from any region code simply do `self.page.click_and_wait_for_load`

Reply:
@TomWoodward I tried to do what you suggest. But I am probably missing something and couldn't get a proper result.